web_widget_image_webcam/static/src/xml/web_widget_image_webcam.xml

```diff
@@ -0,0 +1,42 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<!-- Copyright 2016 Siddharth Bhalgami <siddharth.bhalgami@techreceptives.com>
+     License LGPL-3.0 or later (http://www.gnu.org/licenses/lgpl). -->
+<template>
+    <t t-extend="FieldBinaryImage">
+        <t t-jquery=".oe_form_field_image_controls" t-operation="after">
+            <div class="oe_form_field_image_web_cam_controls oe_edit_only">
+              <a class="fa fa-eye fa-1g col-md-4 oe_form_binary_file_web_cam"
+                 data-toggle="modal" data-target="#WebCamModal" title="WebCam"></a>
+            </div>
+            <div id="WebCamModal" class="modal fade">
+                <div class="modal-dialog modal-lg">
+                    <div class="modal-content">
+                        <div class="modal-header">
+                            <button type="button" class="close" data-dismiss="modal">x</button>
+                            <h4 class="modal-title">WebCam Booth</h4>
+                        </div>
+                        <div class="modal-body">
+                            <div class="row">
+                                <div class="col-md-12">
+                                    <div class="col-md-5 live_webcam_outer_div">
+                                        <div id="live_webcam" style="width:320px; height:240px;"></div>
+                                    </div>
+                                    <div class="col-md-2 mt64" style="text-align: center;">
+                                        <span><i class="fa fa-angle-right fa-8x"/></span>
```

Comment:
Use `<div>` (not `<i>` or `<span>`) for fontawesome icons. These inline elements are included in translations and can result in false positives when outputting the translation template.